app/models/null_git_hub_user.rb

```diff
@@ -6,4 +6,8 @@ def html_url
   def login
     'ghost'
   end
+
+  def name
+    'ghost'
```

Comment:
I prefer to use "Deleted user" here as that's the name that is associated with https://github.com/ghost and makes more sense when the user sees that.